README.md

```diff
@@ -162,7 +168,35 @@ To cite Concrete in academic papers, please use the following entry:
   note={\url{https://github.com/zama-ai/concrete}},
 }
 ```
+### Contributing 
+
+There are two ways to contribute to Concrete. You can:
+- [Open issues](https://github.com/zama-ai/concrete/issues/new/choose) to report bugs and typos, or to suggest new ideas
+- Request to become an official contributor by emailing hello@zama.ai.
+
+Becoming an approved contributor involves signing our Contributor License Agreement (CLA)). Only approved contributors can send pull requests (PRs), so get in touch before you do!
+
+Additionally, you can contribute to advancing the FHE space with Zama by participating in our [Bounty Program and Grant Programs](https://github.com/zama-ai/bounty-and-grant-program)! 
+<br></br>
+ 
+### License
+This software is distributed under the **BSD-3-Clause-Clear**. [Check for more details](https://github.com/zama-ai/concrete?tab=License-1-ov-file)  
```

Comment:
The link is ok? shouldn't it be https://github.com/zama-ai/concrete/blob/main/LICENSE.txt?

Reply:
Indeed it's better. 
Replaced ✅